### PR TITLE
Adding check for token split values

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
fixes: https://github.com/wso2/apk/issues/1661

### CODE REVIEW

File: gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java [hunk 1]
```diff
@@ -246,6 +246,9 @@ public static SignedJWTInfo getSignedJwt(String accessToken,String organization)
     public static boolean isExpired(String token) {
 
         String[] splitToken = token.split("\\.");
+        if (splitToken.length != 3) {
```
Comment (AmaliMatharaarachchi) [inline gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java:249]: how can it reach this line? we have a can authenticate method in apikey and jwt authenticators. there we check whther the token is in right format. i think this check has to be added from that level, invalid tokens shouldn't reach this code

Comment (CrowleyRajapakse) [reply gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java:249]: this ArrayIndexOutOfBoundsException occurs in the next line intermittently, so the previous validations seems to be unaffected in those intermittent scenarios

Comment (AmaliMatharaarachchi) [reply gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java:249]: i think we need to fix from that level too

Comment (CrowleyRajapakse) [reply gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java]: fixed

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**